docs/native-components-android.md

```diff
@@ -44,31 +65,71 @@ public class ReactImageManager extends SimpleViewManager<ReactImageView> {
 }
 ```
 
+</TabItem>
+</Tabs>
+
 ### 2. Implement method `createViewInstance`
 
 Views are created in the `createViewInstance` method, the view should initialize itself in its default state, any properties will be set via a follow up call to `updateView.`
 
+<Tabs groupId="android-language" defaultValue={constants.defaultAndroidLanguage} values={constants.androidLanguages}>
+<TabItem value="kotlin">
+
+```kotlin
+  override fun createViewInstance(context: ThemedReactContext) =
+      ReactImageView(context, Fresco.newDraweeControllerBuilder(), null, mCallerContext)
+```
+
+</TabItem>
+<TabItem value="java">
+
 ```java
   @Override
   public ReactImageView createViewInstance(ThemedReactContext context) {
     return new ReactImageView(context, Fresco.newDraweeControllerBuilder(), null, mCallerContext);
   }
 ```
 
+</TabItem>
+</Tabs>
+
 ### 3. Expose view property setters using `@ReactProp` (or `@ReactPropGroup`) annotation
 
-Properties that are to be reflected in JavaScript needs to be exposed as setter method annotated with `@ReactProp` (or `@ReactPropGroup`). Setter method should take view to be updated (of the current view type) as a first argument and property value as a second argument. Setter should be declared as a `void` method and should be `public`. Property type sent to JS is determined automatically based on the type of value argument of the setter. The following type of values are currently supported: `boolean`, `int`, `float`, `double`, `String`, `Boolean`, `Integer`, `ReadableArray`, `ReadableMap`.
+Properties that are to be reflected in JavaScript needs to be exposed as setter method annotated with `@ReactProp` (or `@ReactPropGroup`). Setter method should take view to be updated (of the current view type) as a first argument and property value as a second argument. Setter should be declared as a `void` method in Java or with return type `Unit` (or omitted) in Kotlin and should be `public`. Property type sent to JS is determined automatically based on the type of value argument of the setter. The following type of values are currently supported (in Java): `boolean`, `int`, `float`, `double`, `String`, `Boolean`, `Integer`, `ReadableArray`, `ReadableMap`. The corresponding types in Kotlin are `Boolean`, `Int`, `Float`, `Double`, `String`, `ReadableArray`, `ReadableMap`.
```

Comment:
I wasn't quite sure how to formulate this section. Especially the `void` / `Unit` part. Suggestions are appreciated!

It might be overly explicit to list all the types as well?

Reply:
Added a suggestion on how to improve it

Reply:
I formulated it slightly different from your suggestion. Let me know if you prefer yours 👍